Documentation/BuildInstructionsLadybird.md

```diff
@@ -1,6 +1,7 @@
 # Ladybird browser build instructions
 
-**NOTE:** The Ladybird cross-platform web browser project is now separate from SerenityOS, and is now actively developed in the [LadybirdBrowser/ladybird](https://github.com/LadybirdBrowser/ladybird) repository.
+> [!NOTE]
+> This is the SerenityOS version of the Ladybird browser. It is synced with the "upstream" version on a best-effort basis, and feature parity between the two is not guaranteed. See the [LadybirdBrowser/ladybird](https://github.com/LadybirdBrowser/ladybird) repository for bug tracking and pull requests.
```

Comment:
I'd write this more along the lines of "The ladybird web browser project has moved into its own repository at <link>. priority is that serenityos's Browser application and serenityos's LibWeb is very polished and functional. We're keeping the pre-fork version of ladybird in the serenityos repository to aid development of serenityos's libweb. The version of ladybird in the serenityos repository is not meant to be a polished browser in host OSs other than serenityos". (…except with more polished wording.) WDYT?

Reply:
Hm. Your version is a bit loaded right now. How about "The Ladybird web browser project has moved to XYZ, this version is kept as a developer convenience for the testing of LibWeb and LibJS libraries included with SerenityOS"?

Reply:
That sounds great!